tests/queries/.gitignore

```diff
@@ -1,2 +0,0 @@
-*.stderr
-*.stdout
```

Comment:
Is this accidental change?
FYI I've restored it in the main `.gitignore`, here - https://github.com/ClickHouse/ClickHouse/pull/25033/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R17